devtools/gradle/gradle.properties

```diff
@@ -1 +0,0 @@
-version = 999-SNAPSHOT
```

Comment:
Have you tried building the plugin the gradle way?

Reply:
Also if you import it into the IDE as a Gradle project, won't it complain that the version is missing?

Reply:
Yes, you're right, it won't work when building using Gradle. Since the release uses Maven, I guess the `-Pversion` should override this value, right? In this case I'll add the properties file back but keep the parameter in the pom.xml, WDYT?

Reply:
I'll let you test and confirm that. BTW, I am not sure what the actual issue you fixing here.